### ISSUE TITLE
onItemSelect event not fires

### ISSUE THREAD

**Issue by tkuzyna-neurosys** [namespace-ee/react-calendar-timeline#137]
Hello

I have a problem with selecting items on timeline. Demo app has the same problem. Please see video
https://www.dropbox.com/s/d8oe25xd33nyfiy/rec_timeline.webm?dl=0
Is there any solution?

I'm testing on Windows 10, Google Chrome v59.0.3071.115 

Best regards,
Taras Kuzyna

**Comment by rafsawicki** [comment]
I can confirm - the same issue happens both on demo and on our internal project. Tested today on latest Chrome (same version as above).

**Comment by mariusandra** [comment]
Hey, I'm using the same Chrome on a Mac (Sierra, 10.12.5) and it works just fine. Can anyone with this issue clone this project, run `yarn start` and try to debug from there? @tkuzyna-neurosys @rafsawicki ?

**Comment by rafsawicki** [comment]
I cloned a repo and started a project. While I wasn't able to build it (`yarn` command failed on building css, maybe due to OS being windows?) I managed to run it with the command you provided. What can I do from there?

As a starter I attached a click event handler at document level. If anything was stopping propagation of those events, an amount of "Selected" log messages and mine should match, but that's not the case. It looks like calendar is just not picking them up.
![image](https://user-images.githubusercontent.com/2313294/28475075-353389c0-6e4b-11e7-9cbb-853539ce988e.png)

**Comment by mariusandra** [comment]
Hey, when looking at the code for the `<Item>` component, it seems the click is handled with `onMouseDown` and `onMouseUp`. I'd recommend adding some `console.log` statements there to see if the right actions are fired.

It is possible that under some situations the `onMouseDown` event gets replaced with `onTouchStart`... or the `onMouseUp/Down` doesn't fire as the browser thinks we started to drag instead of clicking... or somehow the `.on('tap')` in `mountInteract()` triggers instead of some of the mouse events.

Hard to say. I would just add `console.log` statements everywhere and see what the fuzz is about :).

Also, what error did you get with the CSS?

**Comment by rafsawicki** [comment]
I added some basic logging as you suggested. It looks like value of `this.state.interactMounted` is not set back to `false` sometimes, which prevents firing `actualClick` function. [react-calendar-timeline-logs.txt](https://github.com/namespace-ee/react-calendar-timeline/files/1202614/react-calendar-timeline-logs.txt)

I also tested this problem in Firefox 54.0.1. It works there without any issues.

On Chrome I do get an error on console that is not present on Firefox, although it doesn't seem to be related at first glance:
```
Uncaught TypeError: event.target[matches] is not a function
    at HTMLDocument.<anonymous> (context-menu-content-scripts.js:23)
```

As for error in CSS - that's something I didn't checked correctly last time. It's the entire build command that simply won't work with with windows command line (different syntax and lack of `sed` command). It is building nicely on Ubuntu though.
